python/src/FunctionalChaosAlgorithm_doc.i.in

```diff
@@ -181,14 +261,14 @@ Define the evaluation strategy of the  coefficients:
 
 >>> samplingSize = 50
 >>> experiment = ot.MonteCarloExperiment(distribution, samplingSize)
->>> X = experiment.generate()
->>> Y = model(X)
+>>> inputSample = experiment.generate()
+>>> outputSample = model(inputSample)
 >>> projectionStrategy = ot.LeastSquaresStrategy()
 
 Create the chaos algorithm:
 
->>> algo = ot.FunctionalChaosAlgorithm(X, Y, distribution, adaptiveStrategy,
-...                                    projectionStrategy)
+>>> algo = ot.FunctionalChaosAlgorithm(inputSample, outputSample, distribution, \
```

Comment:
Maybe rename "inputSample" as "x" and "outputSample" as "y" to save space and allow this to fit on a single line?

Reply:
These are the names used in the description of the arguments of the constructor of the class. These names cannot be changed because they must match the names used in the C++. This is because SWIG uses these names when generating the part of the API doc which is automatically generated. Hence, I think that this example is better as is.

Reply:
Then I think you should at least align the arguments vertically.

Reply:
That is true. I extracted the code and formatted it using black. Fixed at edf67cfd0